src/hisp/festim_models/mb_model.py

```diff
@@ -749,3 +753,130 @@ def make_DFW_mb_model(
     my_model.settings.stepsize = F.Stepsize(initial_value=1)
 
     return my_model, quantities
+
+
+def calculate_temperature_W(x: float | NDArray, heat_flux, coolant_temp, thickness):
+    T_surface = 1.1e-4 * heat_flux + coolant_temp
+    T_rear = 2.2e-5 * heat_flux + coolant_temp
+    a = (T_rear - T_surface) / thickness
+    b = T_surface
+    return a * x + b
+
+
+def calculate_temperature_B(heat_flux, coolant_temp):
+    T_rear_tungsten = (
+        2.2e-5 * heat_flux + coolant_temp
+    )  # boron layers based off of rear temp of W mbs
+    return 5e-4 * heat_flux + T_rear_tungsten
```

Comment:
@kaelyndunnell I plan to add documentation to these functions. Can you give me the references for these models?

Reply:
Also, what is the justification of the boron temperature?  why 5e-4 * heat_flux + rear temperature?

Reply:
used this paper for tungsten temp model: 10.1088/1741-4326/ac2abc